File: backend/app/api/health.py
from fastapi import APIRouter
from datetime import datetime

router = APIRouter(prefix="/health", tags=["Health"])


@router.get("")
def check_health():
    return {
        "status": "healthy",
        "service": "TrafficVision AI Core API",
        "version": "1.0.0",
        "timestamp": datetime.utcnow().isoformat(),
        "database_connections": {
            "postgresql": "configured",
            "mongodb": "configured",
            "redis": "configured"
        }
    }

File: backend/src/routes/health.js
const express = require('express');
const router = express.Router();

router.get('/health', (req, res) => {
  res.json({
    status: 'healthy',
    service: 'TrafficVision AI Core Express API',
    runtime: `Node.js ${process.version}`,
    version: '1.0.0',
    timestamp: new Date().toISOString(),
    database_connections: {
      postgresql: 'configured',
      mongodb: 'configured',
      redis: 'configured'
    }
  });
});

module.exports = router;
